index.js

```diff
@@ -48,6 +49,11 @@ module.exports = opts => {
 	});
 
 	app.on('ready', () => {
+		// activate devtron for the user if they have it installed
+		try {
```

Comment:
I think it would be better to provide an option to activate it, so you can remove the `try` block